collector/nvme_linux.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2021 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build linux
+// +build !nonvme
+
+package collector
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/procfs/sysfs"
+)
+
+const (
+	subsystem = "nvme"
```

Comment:
That's a package wide const so 'subsystem' isn't specific enough. I'd just keep hardcoding it as you were before

Reply:
Okay. Changed to hardcode it, since it is used only in one place.